cirq-ft/requirements.txt

```diff
@@ -0,0 +1,5 @@
+attrs
+cachetools
+ipywidgets
+nbconvert
```

Comment:
Some of these requirements should be dev-only requirements, correct?  (Ok to fix later if we create an issue).

Reply:
Yes, tracked in https://github.com/quantumlib/Cirq/issues/6141